cplusplus/vips-operators.cpp

```diff
@@ -1,156 +1,156 @@
 // bodies for vips operations
-// Mon 11 Jun 14:28:56 BST 2018
+// Fri 01 Feb 21:18:25 2019
 // this file is generated automatically, do not edit!
 
-void VImage::system( char * cmd_format , VOption *options )
+void VImage::system( char *cmd_format, VOption *options )
 {
-    call( "system" ,
-        (options ? options : VImage::option()) ->
-            set( "cmd-format", cmd_format ) );
+    call( "system",
+        (options ? options : VImage::option())->
+            set( "cmd_format", cmd_format ) );
```

Comment:
The old one (`cmd-format`) seems to be incorrect. See: https://github.com/libvips/libvips/blob/10c4831a709063189d67b7d6076b1b26bf62b2b2/libvips/iofuncs/system.c#L289

Reply:
I think it's harmless, see eg. https://github.com/libvips/libvips/blob/master/cplusplus/vips-operators.cpp#L1244 but I guess needs testing.

Reply:
Hopefully it will be automatically converted from `-` to `_`. Your example also has an underscore:
https://github.com/libvips/libvips/blob/10c4831a709063189d67b7d6076b1b26bf62b2b2/libvips/create/mask_butterworth.c#L94

I'm going to test this.

Reply:
Just tested this, and it seems to makes no difference (so both `_` or `-` will work). Tested with:
```cpp
// g++ -g -Wall example.cc `pkg-config vips-cpp --cflags --libs`

#include <vips/vips8>
#include <cassert>

auto main(int argc, const char *argv[]) -> int {
    if (vips_init(argv[0]) != 0) {
        vips_error_exit(nullptr);
    }

    auto im = vips::VImage::gaussmat(1, 0.1);
    assert(im.width() == 5);
    assert(im.height() == 5);
    assert(im.bands() == 1);
    assert(im.format() == VIPS_FORMAT_DOUBLE);
    assert(im.max() == 20.0);

    auto total = im.avg() * im.width() * im.height();
    auto scale = im.get_double("scale");
    assert(total == scale);

    auto p = im.getpoint(im.width() / 2, im.height() / 2);
    assert(p[0] == 20.0);

    vips_shutdown();
}
```